execution/utility.sh

```diff
@@ -1262,8 +1262,21 @@ function manage_ecs_account_settings() {
   local setting="$1"; shift
   local state="$1"; shift
 
+
+  info "Configuring ecs account setting - ${region} - ${setting} - ${state}"
+  # Remove settings applied at the principal level to ensure defaults are respected
+  principal_setting="$(aws --region ap-southeast-2 ecs list-account-settings --name "${setting}" --query 'settings[].name' --no-effective-settings  | jq -r '.[]')"
```

Comment:
If we already have the `${region}` is it necessary here to hardcode the region on the CLI argument?

Reply:
Gah! good catch, that would have been a copy and paste error from local testing.